fe/fe-core/src/main/java/com/starrocks/load/OlapDeleteJob.java

```diff
@@ -405,16 +404,9 @@ public boolean commitImpl(Database db, long timeoutMs) throws StarRocksException
     @Override
     protected List<TabletCommitInfo> getTabletCommitInfos() {
         List<TabletCommitInfo> tabletCommitInfos = Lists.newArrayList();
-        TabletInvertedIndex invertedIndex = GlobalStateMgr.getCurrentState().getTabletInvertedIndex();
-        for (TabletDeleteInfo tDeleteInfo : getTabletDeleteInfo()) {
-            for (Replica replica : tDeleteInfo.getFinishedReplicas()) {
-                // the inverted index contains rolling up replica
-                Long tabletId = invertedIndex.getTabletIdByReplica(replica.getId());
-                if (tabletId == null) {
-                    LOG.warn("could not find tablet id for replica {}, the tablet maybe dropped", replica);
-                    continue;
-                }
-                tabletCommitInfos.add(new TabletCommitInfo(tabletId, replica.getBackendId()));
+        for (TabletDeleteInfo tabletDeleteInfo : getTabletDeleteInfo()) {
+            for (Replica replica : tabletDeleteInfo.getFinishedReplicas()) {
+                tabletCommitInfos.add(new TabletCommitInfo(tabletDeleteInfo.getTabletId(), replica.getBackendId()));
```

Comment:
this doesn't look like identical code change, need to double check that the caller can handle the case when the tablet id doesn't exist in the TabletCommitInfo.

Reply:
The code here will be checked again externally to see if the tablet still exists.